.rubocop/strict.yml

```diff
@@ -0,0 +1,23 @@
+# Inherit from TODO here to make sure we enforce the rules below (and TODO is ignored)
+inherit_from:
+  - ../.rubocop_todo.yml
+
+Lint/Debugger: # Remove any `binding.pry`
+  Enabled: true
+  Exclude: []
+
+RSpec/Focus: # Require full spec run on CI
+  Enabled: true
+  Exclude: []
+
+Rails/Output: # Remove any `puts` debugging
+  Enabled: true
+  Exclude: []
```

Comment:
Doesn't that conflict with our use of `puts` in `lib/tasks` and `lib/mastodon/migration_helpers.rb`?

Reply:
I think the rule by default does not apply to `.rake`, only to `.rb`.

The migration helpers specifically I think we have completely disabled from all of rubocop. I've been meaning to at some point extract the portion of this which we pulled in from gitlab and put it somewhere in vendor, and leave the portion we wrote where it is, and enable rubocop for it.